### ISSUE TITLE
Fix CssSyntaxError for single-comment comments in side at-rule values

### ISSUE THREAD

**Issue by DemianX0** [stylelint/stylelint#4617]
> Clearly describe the bug

There is a false positive / parse error inside single-line comments for sole quotes.

> Which rule, if any, is the bug related to?

single-line comments\
`CssSyntaxError`

Related: #4616, [#4298](https://github.com/stylelint/stylelint/issues/4298#issuecomment-590111620), #4049, #3975

> What code is needed to reproduce the bug?

Demo:
[single quote](https://stylelint.io/demo#N4Igxg9gJgpiBcIACAzFBaAzgQwHafgB1cACEzAT0wBcYBbdAVwEsAaEgeg5MJABkYANxgAbEgBYA5JhIADSjXpNmskoOwjGMXiQAKAQQDKh4mQW0GLdlxIDhYqTPlULy1es0wyJAMKZMhhS41NgAHgCiAE6REJEk2NTkzLgA5iJeAI6MELSm5HiYWDCRzCgA3MQgrCCUwWEIIOn+VeAQuCjMKQ3AeWS8kYxNvPAkPaTeAL7EEyATQA),
[double quote](https://stylelint.io/demo#N4Igxg9gJgpiBcIACAzFBaAzgQwHafgB1cACEzAT0wBcYBbdAVwEsAaEgeg5MJABkYANxgAbEgBYA5JhIADSjXpNmskoOwjGMXiQAKAQQDKh4mQW0GLdlx78hoidLnmlLVes0wyAYUyZDFLjU2AAeAKIAThEQESTY1CRQEIwARiJeAI6MELSm5HiYWDARzCgA3MQgrCCUQaEIIOl+VeAQuCjMAOYNwHlkvBGMTbzwJL2kZCQAvsRTIFNAA)

```less
@ff-sans:
  system-ui, // "Level 4's `system-ui` value" PASS
  system-ui, // Level 4's `system-ui` value   CssSyntaxError at single quote
  sans-serif;

@ff-sans:
  system-ui, // "Level 4's `system-ui` value" PASS
  system-ui, // "Level 4's `system-ui` value  CssSyntaxError at double quote
  sans-serif;
```

> What stylelint configuration is needed to reproduce the bug?

default / any

> Which version of stylelint are you using?

`13.2.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

`grunt stylelint`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Less. No single-line comments in CSS.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:
```
3:24  Unclosed string (CssSyntaxError)
```

**Comment by jeddy3** [comment]
Thanks again for another informative report and for using the template.

Please consider [contributing to the upstream syntax](https://stylelint.io/developer-guide/syntaxes) to stylelint [uses to support Less](https://stylelint.io/about/syntaxes).

**Comment by jeddy3** [comment]
Closing as the [`syntax` option was removed](https://github.com/stylelint/stylelint/issues/5289) for the [14.0.0 release](https://github.com/stylelint/stylelint/issues/5205). (See the [migration guide](https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-14.md#syntax-option-and-automatic-inferral-of-syntax).)

Please consider fixing the issue upstream as the syntax is no longer bundled by Stylelint.
